From fbe2c2e2c76e52935c82c89d9b04a15560988f9b Mon Sep 17 00:00:00 2001
From: Deep Chatterjee <deep.chatterjee@ligo.org>
Date: Thu, 4 May 2023 11:38:21 -0400
Subject: [PATCH] remove lalsuite

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 46d0da4..dad515b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -23,7 +23,6 @@ python = ">=3.9"
 scikit-learn = "==1.2.1" 
 astropy = ">=5.2.1"
 pandas = ">=1.5.2"
-lalsuite = "^7.0"
 h5py = ">=3.7.0"
 
 [tool.poetry.dev-dependencies]
-- 
2.25.1

