From 94a4a9d8bf1c437573248bd38463aaf8d7dd26d1 Mon Sep 17 00:00:00 2001
From: Deep Chatterjee <deep.chatterjee@ligo.org>
Date: Sat, 5 Oct 2024 10:01:59 -0400
Subject: [PATCH] remove lalsuite for conda build

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 11c6a77..5b618d3 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -27,7 +27,6 @@ python = ">=3.10"
 scikit-learn = "==1.5.1" 
 astropy = "^6.0"
 pandas = "^2.2"
-lalsuite = "^7.23"
 h5py = "^3.11"
 numpy = "*"
 
-- 
2.25.1

