From 5bfc06b1ce9232a95aa3e0e7d76df5df21de4d97 Mon Sep 17 00:00:00 2001
From: Deep Chatterjee <deep.chatterjee@ligo.org>
Date: Fri, 16 Jun 2023 09:39:21 -0400
Subject: [PATCH] remove lalsuite patch

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index df73e83..e353d7d 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -23,7 +23,6 @@ python = ">=3.9"
 scikit-learn = "==1.2.1" 
 astropy = ">=5.2.1"
 pandas = ">=1.5.2"
-lalsuite = "^7.0"
 h5py = ">=3.7.0"
 
 [tool.poetry.dev-dependencies]
-- 
2.25.1

